hw12.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+import json
+import sys
+
+with open('/phonebook', 'w+') as file:

```

Comment:
It is not the best practice to open a file for the whole time program is running. I would suggest you move the main loop outside the context manager block and open a file in places where you need it:
```
with open('/phonebook', 'w+') as file:
        phonebook = json.loads(file.read())

while True:
...
...

        if command == 'add':
            if phonebook.get(name) is None:
                phonebook[name] = phone
                with open('phonebook', 'w+') as file:
                    file.write(json.dumps(phonebook))

Reply:
The same for `delete` command